modules/end-user-guide/pages/troubleshooting-slow-workspaces.adoc

```diff
@@ -64,7 +64,7 @@ To check that, open Developer Tools in the browser and check for `vendors` in th
 
 Providing enough CPU resources::
 +
-Plug-ins consume CPU resources. For example, when a plugin provides IntelliSense features, adding more CPU resources may lead to better performance.
+Plug-ins consume CPU resources. For example, when a plugin provides IntelliSense features, adding more CPU resources can lead to better performance.
```

Comment:
:)
```suggestion
Plugins consume CPU resources. For example, when a plugin provides IntelliSense features, adding more CPU resources can lead to better performance.
```

Reply:
`can lead to better performance.` > `improve performance.`